default-plugins/strider/src/state.rs

```diff
@@ -50,38 +52,89 @@ impl State {
 pub enum FsEntry {
     Dir(PathBuf, usize),
     File(PathBuf, u64),
+    DisplayDir(PathBuf),
 }
 
 impl FsEntry {
     pub fn name(&self) -> String {
         let path = match self {
             FsEntry::Dir(p, _) => p,
             FsEntry::File(p, _) => p,
+            FsEntry::DisplayDir(p) => p,
         };
-        path.file_name().unwrap().to_string_lossy().into_owned()
+        match self {
+            FsEntry::File(..) | FsEntry::Dir(..) => {
+                // only use the filename
+                path.file_name().unwrap().to_string_lossy().into_owned()
+            },
+            FsEntry::DisplayDir(..) => {
+                // use full path, but we need to remove the host part
+                let path = path.to_string_lossy().into_owned().replace("/host", "");
+                ".".to_string() + &path
```

Comment:
Think this can be a bit simpler:
```suggestion
                path.to_string_lossy().replace("/host", "")
```